source/index.html.md

```diff
@@ -19,221 +15,821 @@ search: true
 
 # Introduction
 
-Welcome to the Kittn API! You can use our API to access Kittn API endpoints, which can get information on various cats, kittens, and breeds in our database.
+> To test your connection to our API please use
 
-We have language bindings in Shell, Ruby, and Python! You can view code examples in the dark area to the right, and you can switch the programming language of the examples with the tabs in the top right.
+```shell
+curl https://cerego.com/api/echo 
+    -d '{"echo":"Test"}' 
+    -X POST 
+    -H "Content-Type: application/json"
+```
 
-This example API documentation page was created with [Slate](https://github.com/lord/slate). Feel free to edit it and use it as a base for your own API's documentation.
+> Please note that all other endpoints require authentication via a Bearer Token.
 
-# Authentication
+Welcome to the Cerego API! You can use our API to access Cerego endpoints, which can get information on about courses, assignments, users, and more.
 
-> To authorize, use this code:
+All V3 endpoints follow the [JSON API](http://jsonapi.org/) standard. Please take note how objects within a certain resource are in the includes section and -'s are used to denote spaces.
 
-```ruby
-require 'kittn'
+Cerego provides 3 different environments, 2 for testing and QA, as well as our primary production environment
 
-api = Kittn::APIClient.authorize!('meowmeowmeow')
-```
+`https://cerego.com/` - Production <br>
+`https://stable.cerego.com/` - Stable (Used for QA) <br>
+`https://testing.cerego.com/` - Testing (Used for heavy testing and unreleased features)
```

Comment:
Don't know if we need to tell external people about this since it's pretty unstable.

Reply:
I'm okay either way, would you prefer removing it?

Reply:
I do kind of like the idea that people can use testing and stable to test the API before making their first calls that modify actual production data.

Reply:
I disagree. I don't think we want to be debugging stable/testing except for privileged partners like Cengage, and even them we don't want on testing. They should create their own testing data on prod to test their API with.

Reply:
Fair enough, I've removed all references to testing/stable